corehq/apps/accounting/migrations/0043_add_web_user_to_payment_methods.py

```diff
@@ -0,0 +1,317 @@
+# -*- coding: utf-8 -*-
+from south.db import db
+from south.v2 import DataMigration
+from django.db import connection
+
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        sql = """
+            UPDATE accounting_paymentmethod
```

Comment:
I think doing it in SQL is fine. maybe just throw in a comment stating why it was done so that future people aren't thrown off by it.